components/form-group/src/stories.js

```diff
@@ -11,9 +11,8 @@ import ReadMe from '../README.md';
 const stories = storiesOf('Form/FormGroup', module);
 stories.addDecorator(withKnobs);
 
-stories.add(
-  'Component default',
-  WithDocsCustom(ReadMe, () => (
+stories.add('Component default', () =>
+  WithDocsCustom(ReadMe)(() => (
```

Comment:
should this not be `withDocsCustom`? :wink:

(to be honest, i was confused by this commit, because i'd only very recently added in this story, using this slightly different way of using `withDocsCustom`, and was sure that i'd used the API correctly.  as it goes, i had done things correctly - the underlying `withDocs` API had just changed since, and our API has changed to match it.)